.env.sample

```diff
@@ -1,22 +1,26 @@
 NODE_ENV=development
 PORT=3004
 #HOSTNAME for AuthService
-HOST_HOSTNAME=
+HOST_HOSTNAME=localhost
 
+## Database settings
 DB_USERNAME=
 DB_PASSWORD=
 MYSQL_ROOT_PASSWORD=
 DB_PORT=3307 # This values maps mysql docker container to your local machine, if you already have something listening on port 3307 you will need to update this value to any available port.
 DB_NAME=authservice
-DB_HOSTNAME=auth_mysql_db
+DB_HOSTNAME=auth_mysql_db # If running on local machine change to localhost
```

Comment:
I think this should say - "if app is not running as Docker container, change to localhost"

Reply:
Updated